bingo/EA/DeterministicCrowding.py

```diff
@@ -0,0 +1,48 @@
+"""The "Deterministic Crowding" evolutionary algorithm
+
+This module defines the basis of the "deterministic crowding"
+evolutionary algorithm in bingo analyses. The next generation
+is selected by pairing parents with their offspring and
+advancing the most fit of the two.
+"""
+from bingo.Base.EvolutionaryAlgorithm import EvolutionaryAlgorithm
```

Comment:
lets use relative imports here
```suggestion
from ..Base.EvolutionaryAlgorithm import EvolutionaryAlgorithm
```